monkey/monkey_island/cc/server_utils/island_logger.py

```diff
@@ -63,3 +64,24 @@ def reset_logger():
 
     for handler in logger.handlers:
         logger.removeHandler(handler)
+
+
+def get_log_file_path() -> Optional[Path]:
+    """
+    Finds the log file by finding the logger handlers and checking if one of them is a fileHandler
+    of any kind by checking if the handler has the property handler.baseFilename.
+
+    :return: Log file path
+    """
+
+    logger = logging.getLogger(__name__)
+
+    logger_handlers = logger.parent.handlers
```

Comment:
I'm not sure this is the best way to retrieve the logger path since there could be multiple file handlers. I think the best way to retrieve the file path is the same way it was added. Add a method
```
def get_logger_path(data_dir: Path):
    return data_dir / ISLAND_LOG_FILENAME
```
And then call it from `setup_logging` and from the `_get_log_file_contents`.